docker/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Build for us-docker.pkg.dev/cloud-build-fhir/openmrs/openmrs-reference-application-distro:latest image
+
+FROM openmrs/openmrs-reference-application-distro:nightly
+
+RUN apt-get clean && apt-get update && apt-get install -y dialog apt-utils
+
+RUN apt-get install -y locales
+
+RUN locale-gen
```

Comment:
I would get intermittent errors when loading a page with the following error:

```
“Theme ‘green’: No message found under code ‘favicon’ for locale ‘en’.”
```

I cam across this Talk thread where the person ran the `locale-gen` command to fix. So I installed it and the issue hasn't re-appeared since then:

https://talk.openmrs.org/t/theme-green-no-message-found-under-code-favicon-for-locale-en-gb/30836

Reply:
I see, let's add a comment here about this.